index.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
Can you please add a new line after this, and if you'll squash the commits I can merge this?